website/content/partials/consul-dataplane-upgrade-note.mdx

```diff
@@ -0,0 +1,2 @@
+## Consul Dataplane Compatibility
```

Comment:
```suggestion
### Consul Dataplane Compatibility
```
I was wrong in slack about the headers - this is under "## Changes" so it should be H3.